synapseclient/client.py

```diff
@@ -2039,7 +2041,24 @@ def getTeamMembers(self, team):
         for result in self._GET_paginated('/teamMembers/{id}'.format(id=id_of(team))):
             yield TeamMember(**result)
 
-    def submit(self, evaluation, entity, name=None, team=None, silent=False, submitterAlias=None, teamName=None):
+    def _get_docker_digest(self, entity, docker_tag):
+        '''
+        Get matching Docker sha-digest of a DockerRepository given a Docker tag
+        
+        :param entity:      Synapse id of Docker repository
+        :param docker_tag:  Docker tag
+        :returns: Docker digest matching Docker tag
+        '''
+        docker_tags = self.restGET('/entity/{entityId}/dockerTag'.format(entityId=entity.id)) 
+        docker_digest = None
+        for tag in docker_tags['results']:
```

Comment:
The result of the rest call should be a collection of DockerCommit. Each item in the list should be a docker_commit (not tag) which include the tag and the digest: https://docs.synapse.org/rest/GET/entity/id/dockerTag.html